app/uk/gov/hmrc/tariffclassificationfrontend/views/refer_case_error.scala.html

```diff
@@ -0,0 +1,43 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.tariffclassificationfrontend.config.AppConfig
+@import uk.gov.hmrc.tariffclassificationfrontend.models.Case
+@import uk.gov.hmrc.tariffclassificationfrontend.views.html.includes.main
+@import uk.gov.hmrc.tariffclassificationfrontend.views.html.partials.case_heading
+@import uk.gov.hmrc.tariffclassificationfrontend.views.html.partials.contact_details
+@import uk.gov.hmrc.tariffclassificationfrontend.views.forms.components.RadioOption
+@import uk.gov.hmrc.tariffclassificationfrontend.views.html.forms.components.input_radiogroup
+
+@import uk.gov.hmrc.tariffclassificationfrontend.controllers.routes.AttachmentsController
+@(c: Case)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+
+@main(bodyClasses = None) {
+
+    @case_heading(c)
+
+    <div class="grid-row">
+        <div class="column-full">
+            <h2 class="heading-large mt-0">You must contact the applicant and attach a copy of the email to this case before you refer it</h2>
```

Comment:
Do we need some more IDs for the acceptance tests here?

Reply:
Ash says `headings are fine as is`